src/app/python/core.py

```diff
@@ -245,17 +245,19 @@ def unregister_user_to_all_files(self, user):
     self._add_task(self._task_unregister_user_to_all_files, user)
     self._logger.info("unregister_user_to_all_files task added")
 
-  def file_edit(self, path, changes):
+  def file_edit(self, path, changes, caller):
     """
     Send changes, text added or text removed, to the file
 
     @type path: str
     @type changes: list [Change namedtuple]
+    @type caller: str
 
     @param path: The path of the file in the project tree
     @param changes: Changes to be applied on the file
+    @param caller: The user name
```

Comment:
@vphilippon  And possibly add "The author of the changes" or something like that